### PR TITLE
HAWKULAR-1320 Add a EAP6 jmx exporter

### PR DESCRIPTION
**Pull request by lucasponce**
Adjust scrape config to versioning types

### CODE REVIEW

**Comment by lucasponce** [comment]
There are a couple of PRs on agent and services, but this is the first one to resolve dependencies between them.

**Comment by lucasponce** [comment]
Only this is required https://github.com/hawkular/hawkular-agent/pull/408 services can bump to latest versions of commons and agent when it is done.

**Comment by lucasponce** [comment]
DNM label set, it seems I missed something in the tests.

File: hawkular-inventory-parent/hawkular-inventory-service/src/main/resources/eap-6-jmx-exporter.yml [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+lowercaseOutputLabelNames: true
+whitelistObjectNames: [
+  "org.hawkular.agent:type=platform,subtype=*,name=*",
+  "jboss.as:path=*",
+  "jboss.as:deployment=*,subsystem=ejb3,stateless-session-bean=*",
+  "jboss.as:deployment=*,subsystem=ejb3,stateful-session-bean=*",
+  "jboss.as:deployment=*,subsystem=ejb3,message-driven-bean=*",
+  "jboss.as:deployment=*,subsystem=ejb3,singleton-bean=*",
+  "jboss.as:deployment=*,subsystem=undertow",
+  "jboss.as:deployment=*,subsystem=undertow,servlet=*",
```
Comment (jmazzitelli) [inline hawkular-inventory-parent/hawkular-inventory-service/src/main/resources/eap-6-jmx-exporter.yml]: EAP6 does not have undertow subsystem - it is called "web"

File: hawkular-inventory-parent/hawkular-inventory-service/src/main/resources/eap-6-jmx-exporter.yml [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  "jboss.as:deployment=*,subdeployment=*,subsystem=ejb3,stateless-session-bean=*",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=ejb3,stateful-session-bean=*",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=ejb3,message-driven-bean=*",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=ejb3,singleton-bean=*",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=undertow",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=undertow,servlet=*",
+  "jboss.as:subsystem=messaging-activemq,server=*,jms-queue=*",
+  "jboss.as:subsystem=messaging-activemq,server=*,jms-topic=*",
```
Comment (jmazzitelli) [inline hawkular-inventory-parent/hawkular-inventory-service/src/main/resources/eap-6-jmx-exporter.yml]: I *think* the messaging subsystem is hornetq in EAP6 - need to use the correct names here

**Comment by lucasponce** [comment]
@jmazzitelli I missed the changes on the eap6 file.
Fixed now.

**Comment by lucasponce** [comment]
Still debugging an issue in P8S side, probably some bad config on the jmx config. I will remove the DNM label when ready.

**Comment by jmazzitelli** [comment]
this looks good

**Pull request merged**